src/templates/__tests__/LazyTemplateInterface.spec.js

```diff
@@ -0,0 +1,47 @@
+import LazyTemplateInterface from '../LazyTemplateInterface';
+import MockTemplateFactory from './MockTemplateFactory';
+import { constructWithGetters } from '../utils';
+import mockProvider from './mockProvider';
+
+describe('LazyTemplateInterface', () => {
+  it('throws when setting an undefined template', () => {
+    expect(() => constructWithGetters(LazyTemplateInterface, {
+      checkbox: MockTemplateFactory,
+      checkboxGroup: MockTemplateFactory,
+      dateTextField: MockTemplateFactory,
+      dropDown: MockTemplateFactory,
+      formPage: MockTemplateFactory,
+      radio: MockTemplateFactory,
+      staticPage: MockTemplateFactory,
+      struct: MockTemplateFactory,
+      textArea: MockTemplateFactory,
+      textField: MockTemplateFactory,
+
+      extraField: MockTemplateFactory, // => extra
+    })).to.throw();
+  });
+
+  it('throws when a template is missing', () => {
+    expect(() => constructWithGetters(LazyTemplateInterface, {
+      checkbox: MockTemplateFactory,
+      checkboxGroup: MockTemplateFactory,
+      dateTextField: MockTemplateFactory,
+      dropDown: MockTemplateFactory,
+      formPage: MockTemplateFactory,
+      radio: MockTemplateFactory,
+      staticPage: MockTemplateFactory,
+      struct: MockTemplateFactory,
+      textArea: MockTemplateFactory,
+      // textField: MockTemplateFactory, => missing
```

Comment:
delete

Reply:
I put this here to make it clearer that this is the missing field being tested. Is it not helpful for you?